src/System.Numerics.Vectors/src/System/Numerics/Matrix4x4.cs

```diff
@@ -1779,15 +1806,31 @@ public static Matrix4x4 Transpose(Matrix4x4 matrix)
             return result;
         }
 
+#if netcoreapp
+        private static Vector128<float> Lerp(Vector128<float> a, Vector128<float> b, Vector128<float> t) => 
```

Comment:
It would be nice if this was in a new `internal static class VectorMath` type, to facilitate sharing with `Vector2/3/4` (and the other types, where possible)

Reply:
I wrote a small prototype of a such VectorMath but I am not sure about the fallbacks, could you please take a look? https://gist.github.com/EgorBo/6e0c1d78ee07cfc3fcb52cd3df7d9120